### PR TITLE
Split long running jck targets into sub-groups

### PR DESCRIPTION
**Pull request by Mesbah-Alam**
- Split long running jck targets into sub-groups
- Related : backlog/issues/546

Signed-off-by: Mesbah-Alam <Mesbah_Alam@ca.ibm.com>

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
Test : Grinder/17082

File: jck/compiler.lang/playlist.xml [hunk 1]
```diff
@@ -350,6 +350,286 @@
 			<group>jck</group>
 		</groups>
 	</test>
+	<test>
+	<testCaseName>jck-compiler-lang-EXPR-group1</testCaseName>
+		<disables>
+			<disable>
+				<comment>Disabled on win and ppc64le due to backlog/issues/487. Awaiting some automation improvements, in the interim these targets will be run manually</comment>
+				<platform>ppc64le_linux(_mixed)?|x86-64_windows(_mixed)?</platform>
```
Comment (llxia) [inline jck/compiler.lang/playlist.xml:358]: Please remove `(_mixed)?`. We do not need it anymore.

Comment (Mesbah-Alam) [reply jck/compiler.lang/playlist.xml]: Thanks. I've removed `(_mixed)?` from the playlists associated with this PR here. I'll have a separate PR to remove it from the rest of the tctk playlists.

**Comment by Mesbah-Alam** [comment]
This blocks : https://github.com/adoptium/aqa-tests/pull/2762

**Comment by Mesbah-Alam** [comment]
Test with tck 16 : Grinder/17496/
Test with tck 17 : Grinder/17497/

**Review by llxia — APPROVED** [review]
Thanks @Mesbah-Alam . LGTM

**Review by smlambert — APPROVED** [review]
LGTM

**Pull request merged**